### PR TITLE
image and capture photo location extraction, map popup adjusted

### PR DESCRIPTION
**Pull request by nikkylu**

### CODE REVIEW

File: front-end/src/Login.jsx
Comment (nikkylu) [inline front-end/src/Login.jsx:1]: how can I remove this

**Comment by afknero** [comment]
The map popup looks great, the map successfully navigates to the user's location when permissions are granted, and uploading an image works well. 

There are some issues that should be addressed in the future, mainly submitting a post without completing all required fields. Also, after clearing the location, I was unable to select a new one.

File: package.json [hunk 1]
```diff
@@ -2,5 +2,8 @@
   "devDependencies": {
     "prettier": "^3.3.3",
     "prettier-plugin-tailwindcss": "^0.6.8"
+  },
+  "dependencies": {
```
Comment (afknero) [inline package.json:6]: remove this dependency if it isn't required in the project root

**Comment by nikkylu** [comment]
working on the fix for unable to select a new location

**Pull request merged**